db/migrate/20241014031828_add_deleted_at_to_course_reference_timelines.rb

```diff
@@ -0,0 +1,6 @@
+class AddDeletedAtToCourseReferenceTimelines < ActiveRecord::Migration[7.0]
+  def change
+    add_column :course_reference_timelines, :deleted_at, :datetime
```

Comment:
Why does Course Reference Timeline need soft-delete for course-user deletion?

Reply:
I will revert this one